### DISCUSSION TITLE
Tutorial on enable tap-to-click for touchpad user.

### DISCUSSION THREAD

**Discussion by ahmadraniri** [CuarzoSoftware/Louvre#20] [General]
Hello... So I'm using one of the example compositor based on Louvre (louvre-views). I'm a laptop user with touchpad (obviously). Today I found that we can use tap-to-click touchpad using this [library](https://gitlab.com/warningnonpotablewater/libinput-config). If you need it you can try it. Thanks.

**Comment by ehopperdietzel** [comment]
Haha! This is a really cool hack! Thanks for sharing it. I will see if I can pin it.
Just in case you were wondering, you can access the libinput context struct used by the libinput backend with [`LSeat::inputBackendContextHandle()`](https://cuarzosoftware.github.io/Louvre/class_louvre_1_1_l_seat.html#a19da3003d4204a109769049ca44c0a31) which you can use to configure devices programmatically if required.

**Comment by ahmadraniri** [comment]
This is my config (`libinput.conf`) : 
```
override-compositor=enabled
tap=enabled
dwt=disabled
drag-lock=enabled
```
Just make a config under /etc.
